ksqldb-common/src/main/java/io/confluent/ksql/properties/with/CommonCreateConfigs.java

```diff
@@ -197,6 +198,12 @@ public static void addToConfigDef(
             ConfigDef.Type.INT,
             null,
             Importance.LOW,
+            "Undocumented feature"
```

Comment:
Undocumented?

Is there a reason for it to be undocumented?

Reply:
The initial hunch to keep it undocumented was because we don't want users to set the cleanup policy manually. The policy is inferred by our planner based on the type of source we are creating (stream/table/windowed/non-windowed/C*/C*AS). I have updated the PR and added the config doc copied from kafka